templates/catalog/count_input.html

```diff
@@ -1,7 +1,7 @@
 <div class="input-wrapper">
   <input class="table-count-input text-center js-count-input"
-         value="{{ product.quantity|default:'1' }}" type="number"
-         data-last-count="{{ product.quantity|default:'1' }}"
+         value="1" type="number"
```

Comment:
It will broke `table_form.html`

It would be nice to replace `{{ product.quantity|default:'1' }}` to `{{ quantity }}` and include this template so `{% include 'catalog/count_input.html' with quantity=1 %}`